web/server/models.py

```diff
@@ -173,13 +174,36 @@ class Column(BaseModel):
     description: t.Optional[str]
 
 
+class ModelDetails(BaseModel):
+    owner: t.Optional[str] = None
+    kind: t.Optional[str] = None
+    batch_size: t.Optional[int] = None
+    cron: t.Optional[str] = None
+    stamp: t.Optional[str] = None
+    start: t.Optional[str] = None
+    retention: t.Optional[str] = None
+    storage_format: t.Optional[str] = None
+    time_column: t.Optional[str] = None
+    tags: t.Optional[str] = None
```

Comment:
Should we keep this as a list of strings so it's more flexible?

Reply:
i made it string so UI does not need to do a transformation